dlpy/model.py

```diff
@@ -1009,183 +1009,183 @@ def forecast(self, test_table=None, horizon=1, train_table=None, layer_out=None,
         :class:`CASTable`
 
         """
-            
+
         if horizon > 1:
             self.score_message_level = 'error' #prevent multiple notes in multistep forecast
 
         if self.train_tbl is None:
             self.train_tbl = train_table
-            
+
         if not isinstance(self.train_tbl, TimeseriesTable):
             raise RuntimeError('If the model is not fitted with a TimeseriesTable '+
                                '(such as being imported from other sources), '+
-                               'please consider use the train_table argument ' + 
+                               'please consider use the train_table argument ' +
```

Comment:
You don't have to change it now, but as a note for the future, when you have a series of strings you want appended in Python you don't need the '+' between them.  Python automatically joins them into a single string.